tools/mo/openvino/tools/mo/moc_frontend/check_config.py

```diff
@@ -10,17 +10,35 @@
 
 
 def any_extensions_used(argv: argparse.Namespace):
-    return hasattr(argv, 'extensions') and argv.extensions is not None and len(argv.extensions) > 0 \
```

Comment:
please add a comment here, what kind of extensions are supported: string (for paths) or Extension python objects?

Reply:
For new type extension it can be any class inherited from BaseExtension, if it has Python binding. Added comment.